app/src/main/res/values/strings.xml

```diff
@@ -565,4 +565,7 @@ be temporary, and you can try again later.</li>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
     <string name="onboarding_turbo_mode_button_on">Keep Turbo Mode Enabled</string>
     <string name="onboarding_turbo_mode_button_off">Turn Turbo Mode Off</string>
+
+    <!-- Label for the hint to close the overlay-->
```

Comment:
nit: since we don't extract these hints for l10n afaik, I think the comment is kind of redundant.